lib/index.js

```diff
@@ -363,9 +363,13 @@ class JsonApiSerializer extends Serializer {
 
     return data.map(record => {
       const clone = {}
+      try{
```

Comment:
rather than `try`, I think this would work better:
```js
if (!(reservedKeys.type) in record)
  throw new BadRequestError('The required field "type" is missing.')
```